libs/model/src/thread/GetBulkThread.query.ts

```diff
@@ -194,14 +180,12 @@ export function GetBulkThreads(): Query<typeof schemas.GetBulkThreads> {
           ? t.collaborators.map((c: any) => JSON.parse(c))
           : [];
 
-        const last_edited = getLastEdited(t);
-
         const data: z.infer<typeof schemas.BulkThread> = {
           id: t.thread_id,
           title: t.thread_title,
           url: t.url,
           body: t.body,
-          last_edited,
+          last_edited: t.updated_at,
```

Comment:
1. I don't think `t.updated` exists -> `t.thread_updated`
2. I believe this will show an incorrect timestamp. Edited should show the last _user_ edit but this will show the last _record_ update (e.g. someone commented on the thread and `t.comment_count` is updated).

Reply:
Good catch, although this route isn't used on the front end, I have changed it back to last_edited